libs/base/ui/structure/collapsible/collapsible.component.spec.ts

```diff
@@ -121,4 +121,67 @@ describe('CollapsibleComponent', () => {
       expect(element).toContainElement('summary[tabindex="-1"]');
     });
   });
+
+  describe('when a syncKey is provided', () => {
+    vi.spyOn(globalThis.sessionStorage.__proto__, 'getItem');
+    vi.spyOn(globalThis.sessionStorage.__proto__, 'setItem');
+    let details: HTMLDetailsElement | undefined | null;
+
+    beforeEach(async () => {
+      mockFeatureVersion('1.4');
+      element = await fixture(
+        html`<oryx-collapsible syncKey="foo"></oryx-collapsible>`
+      );
+      details = element.shadowRoot?.querySelector('details');
+      element.scrollIntoView = vi.fn();
+    });
+
+    it('should not load the collapsible state from sessionStorage', () => {
+      expect(globalThis.localStorage.getItem).not.toHaveBeenCalled();
```

Comment:
Are you sure that this is working as expected? If I'm not mistaken need to store reference to the SpyInstance and check its calls, ie:
```
const spy = vi.spyOn(globalThis.sessionStorage.__proto__, 'getItem');
expect(spy).not.toHaveBeenCalled();
```

Reply:
Seems to work, see other tests where we use `expect(globalThis.localStorage.setItem).toHaveBeenCalledWith(`